src/gs-grid.ts

```diff
@@ -95,6 +101,9 @@ export class GsGrid extends HTMLElement {
      */
     private onGridConfigSet(gridConfig: IGridConfig) {
         this.gridConfig = gridConfig;
+
+        // Initialize Cell utils with params.
+        this.cellUtils = new CellUtilities(this.clientWidth);

```

Comment:
Client width should get full width of grid or its parent. If `clientWidth` is not enough, find a way to get best max width possible, either from parent or self.